### PR TITLE
 [SMALLFIX]Add javadoc for the  inner class Resource of  DynamicResourcePoolTest.java 

### PR DESCRIPTION
**Pull request by Lichongjie**

### CODE REVIEW

File: core/common/src/test/java/alluxio/resource/DynamicResourcePoolTest.java [hunk 1]
```diff
@@ -28,15 +28,29 @@
...
+     * @param i the number represent the Resource entity
+     */
     public Resource(Integer i) {
       mInteger = i;
     }
 
+    /**
+     * Set the the number representing Resource entity and return Resource Object.
+     *
+     * @param i
```
Comment (apc999) [inline core/common/src/test/java/alluxio/resource/DynamicResourcePoolTest.java]: ` @param i the number represent the Resource entity` ?

**Comment by yupeng9** [comment]
@Lichongjie there are style issues. You can click the jenkins build link to see the details.

File: core/common/src/test/java/alluxio/resource/DynamicResourcePoolTest.java [hunk 2]
```diff
@@ -28,15 +28,30 @@
...
+   */
   private static final class Resource {
     private Integer mInteger = 0;
     // Threshold for invalid resource.
     private static final int INVALID_RESOURCE = 10;
 
+    /**
+     * Constructor of Resource class.
+     *
+     * @param i the number represent the current capacity of Resource
```
Comment (yupeng9) [inline core/common/src/test/java/alluxio/resource/DynamicResourcePoolTest.java]: represent -> represents

File: core/common/src/test/java/alluxio/resource/DynamicResourcePoolTest.java [hunk 3]
```diff
@@ -28,15 +28,30 @@
 import java.util.concurrent.TimeoutException;
 
 public final class DynamicResourcePoolTest {
+
+  /**
+   * Resource class used to be test DynamicResourcePool.
```
Comment (yupeng9) [inline core/common/src/test/java/alluxio/resource/DynamicResourcePoolTest.java]: used to test

Comment (yupeng9) [inline core/common/src/test/java/alluxio/resource/DynamicResourcePoolTest.java]: also use `@link{DynamicResourcePool}` to reference the class

File: core/common/src/test/java/alluxio/resource/DynamicResourcePoolTest.java [hunk 4]
```diff
@@ -28,15 +28,30 @@
...
+     * Constructor of Resource class.
+     *
+     * @param i the number represent the current capacity of Resource
+     */
     public Resource(Integer i) {
       mInteger = i;
     }
 
+    /**
+     * Set the the number representing current capacity of Resource and return Resource Object.
```
Comment (yupeng9) [inline core/common/src/test/java/alluxio/resource/DynamicResourcePoolTest.java]: Sets the number .. and returns ..

File: core/common/src/test/java/alluxio/resource/DynamicResourcePoolTest.java [hunk 5]
```diff
@@ -28,15 +28,30 @@
...
+     *
+     * @param i the value of member variable represent the current capacity of Resource
```
Comment (yupeng9) [inline core/common/src/test/java/alluxio/resource/DynamicResourcePoolTest.java]: represent -> that represents

File: core/common/src/test/java/alluxio/resource/DynamicResourcePoolTest.java [hunk 6]
```diff
@@ -30,7 +30,8 @@
 public final class DynamicResourcePoolTest {
 
   /**
-   * Resource class used to be test DynamicResourcePool.
+   * Resource class used to test {@link DynamicResourcePool} and referenced by
```
Comment (yupeng9) [inline core/common/src/test/java/alluxio/resource/DynamicResourcePoolTest.java]: It's okay to just use ` Resource class used to test {@link DynamicResourcePool}`.

The remaining part was my explanation :)

**Pull request merged**